docs/operations.md

```diff
@@ -28,6 +28,138 @@ Clients may inspect this specification at run-time so to obey the various limits
 
 For convenience, wfx includes a built-in Swagger UI accessible at runtime via <http://localhost:8080/api/wfx/v1/docs>, assuming default listening host and port [configuration](configuration.md).
 
+### Job Events
+
+Job events provide a notification mechanism that informs clients about certain operations happening on jobs.
+This approach eliminates the need for clients to continuously poll wfx, thus optimizing network usage and client resources.
+Job events can be useful for user interfaces (UIs) and other applications that demand near-instantaneous updates.
+
+#### Architecture
+
+The job events notification system leverages [server-sent events](https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events) (SSE) for its implementation.
+Below is a high-level overview of how the communication flow operates:
+
+```txt
+ ┌────────┐                            ┌─────┐
+ │ Client │                            │ wfx │
+ └────────┘                            └─────┘
+     |                                    |
+     |        HTTP GET /jobs/events       |
+     |-----------------------------------►|
+     |                                    |
+     |                                    |
+     |           Event Loop               |
+ ┌───|────────────────────────────────────|───┐
+ │   |  [Content-Type: text/event-stream] |   │
+ │   |                                    |   │
+ │   |                                    |   │
+ │   |            Push Event              |   │
+ │   |◄-----------------------------------|   │
+ │   |                                    |   │
+ └───|────────────────────────────────────|───┘
+     |                                    |
+     ▼                                    ▼
+ ┌────────┐                            ┌─────┐
+ │ Client │                            │ wfx │
+ └────────┘                            └─────┘
+```
+
+1. The client initiates communication by sending an HTTP `GET` request to the `/jobs/events` endpoint. Clients may also
+   include optional [filter parameters](#filter-parameters) within the request.
+2. Upon receipt of the request, `wfx` sets the `Content-Type` header to `text/event-stream`.
+3. The server then initiates a stream of job events in the response body, allowing clients to receive instant updates.
+
+#### Event Format Specification
+
+The event stream looks as follows:
```

Comment:
Note that this format with `data` and `id` is given by the SSE spec and not our invention.

Reply:
Yes, indeed. I have revised the wording.